addon/components/cookie-banner/style.scss

```diff
@@ -0,0 +1,20 @@
+#cookie-banner {
```

Comment:
Drive by:
You can use `&` instead of `#cookie-banner` to select the parent (in this case the generated component). More info on [the sass ampersand](https://css-tricks.com/the-sass-ampersand/).